sale_timesheet_invoice_description/readme/CONTRIBUTORS.rst

```diff
@@ -6,6 +6,8 @@
   * Manuel Calero
   * Luis D. Lafaurie
 
+
 * `Akretion <https://www.akretion.com>`_:
 
   * Clément Mombereau <clement.mombereau@akretion.com.br>
+  * Dhara Solanki <dhara.solanki@initos.com>
```

Comment:
Incorrect indentation. This way, it's inside Akretion section.

Reply:
👉 I am fixing this.

Thank you for your advice.